src/main/java/com/basiscomponents/db/exportconfig/ColumnConfigurationBuilder.java

```diff
@@ -0,0 +1,51 @@
+package com.basiscomponents.db.exportconfig;
+
+/**
+ * Builder pattern implementation for class ColumnConfiguration
```

Comment:
I think the builder pattern might be a bit too much for this small class. I think a constructor using the fields is enough, especially because the fields are non final and the setter are public

Reply:
I implemented builder pattern foreseeing many more configurations to come. If implemented with just parameterized constructors, this situation would demand either 

- a huge number of constructor overloads (as many config attributes might be optional) or 
- the exposure of the protected access to the empty constructor `ColumnConfiguration()` (meaning user of the classes is gonna have to be trusted with object initialization details)

If you believe the this class with remain small or something else would happen, I can go ahead remove the builder class. Please advise.

Reply:
I would use a Constructor with the necessary fields and public setters for optional fields. But you are right, that this might change in the future. You can leave it like this, I just wanted to know what your thoughts were.

Reply:
Understood. Keeping it the way it is for now.